core/arch/riscv/kernel/entry.S

```diff
@@ -176,7 +170,7 @@ END_FUNC reset_secondary
 DECLARE_KEEP_PAGER reset_secondary
 
 LOCAL_FUNC unhandled_cpu , :
-	j	cpu_idle
+	wfi
```

Comment:
This patch ("core: riscv: implement cpu_idle() function ") is weird:
- `cpu_idle()` already exists (in assembly), the patch it essentially moving it to C (which is OK). The subject needs to be modified accordingly.
- The description mentions "a panicked CPU", but this is `unhandled_cpu()` => "unhandled CPU" would sounds better (what is a panicked CPU at this point?)
- You're replacing `j cpu_idle` with `wfi` which is OK but should be a separate patch, and then explain why `cpu_idle()` is not as good (I suspect the reason is because `mb()` is useless)